main.tf

```diff
@@ -32,5 +32,6 @@ module "lambda_api" {
   ecr_repo                    = var.ecr_repo
   image_tag                   = var.image_tag
   lambda_endpoint_definitions = var.lambda_endpoint_definitions
+  lambda_environment_variables = var.lambda_environment_variables
```

Comment:
Looks like you need to run terraform fmt here before it will work

Reply:
You're so right